http/http.go

```diff
@@ -69,15 +69,22 @@ func ipFromForwardedForHeader(v string) string {
 	return v[:sep]
 }
 
-func ipFromRequest(headers []string, r *http.Request) (net.IP, error) {
+func ipFromRequest(headers []string, r *http.Request, customIP bool) (net.IP, error) {
```

Comment:
I think we must document this feature 🤔 

I suggest simply adding something like this to README (Features section) and index.html (a new paragraph under "Multiple command line HTTP clients..."):

> All endpoints except `/port` can return information about a custom IP address specified via `?ip=` query parameter. 

@mpolden do you have any preference here?

Reply:
sounds good. I've added this, and documented the function. Let me know.

Reply:
Nice! I was thinking to also add the same line to the top of index.html, so that people who visit https://ifconfig.co/ can also discover this feature, what do you think?

Reply:
I'm not sure. I'll defer to @mpolden

Reply:
Yes, feel free to add the paragraph that @maximbaz suggested.

Reply:
:+1: